src/device_trezor/trezor/transport.cpp

```diff
@@ -157,7 +157,11 @@ namespace trezor{
 #define PROTO_HEADER_SIZE 6
 
   static size_t message_size(const google::protobuf::Message &req){
+#if GOOGLE_PROTOBUF_VERSION < 3006001
+    return static_cast<size_t>(req.ByteSize());
```

Comment:
Trivial, but `size_t(req.ByteSize())` (no `static_cast` needed here) is also valid in C++ (constructor syntax). The behavior is also identical.

Reply:
updated